packages/binary-install/index.js

```diff
@@ -97,7 +97,7 @@ class Binary {
 
   run(fetchOptions) {
     if (!this.exists()) {
-      this.install(fetchOptions, true)
+      this.install(fetchOptions, true);
```

Comment:
This was a Prettier change; it looks like `main` is currently failing the Prettier check:

```
λ nix-shell -p nodejs --command "npm run fmt:check"

> fmt:check
> prettier --check **/**/*.js

Checking formatting...
packages/binary-install/index.js
Code style issues found in the above file(s). Forgot to run Prettier?
```